tests/testthat/test-tamper.R

```diff
@@ -0,0 +1,82 @@
+
+context("Tampering")
+
+test_that("We can detect if we are in a pipe", {
+
+  skip(message = "not ready yet for a real test")
```

Comment:
Seems like you need to extract out another function that can be called directly from within a pipe.